packages/builder/src/components/common/CodeEditor/CodeEditor.svelte

```diff
@@ -139,6 +149,46 @@
     }
   }
 
+  // Please write me a function to get the current number of seconds past the hour.
```

Comment:
Don't think we need this comment?

Reply:
Whoops, I left it there so I could copy it when I needed it instead of typing it out each time 😅